### PR TITLE
Add IoT Class to Telegram Integration

### PR DESCRIPTION
**Pull request by milanmeu**
## Proposed change
As requested in #14661 this is to add the IoT class to the Telegram integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: #14661 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/telegram.markdown [hunk 1]
```diff
@@ -4,6 +4,7 @@ description: Instructions on how to add Telegram notifications to Home Assistant
 ha_category:
   - Notifications
 ha_release: 0.7.5
+ha_iot_class: Cloud Polling
```
Comment (frenck) [inline source/_integrations/telegram.markdown:7]: How did you get to this conclusion?

Comment (milanmeu) [reply source/_integrations/telegram.markdown:7]: The telegram notification platform requires the telegram_bot integration to work with.
https://www.home-assistant.io/integrations/telegram_bot/#telegram-notification-platform

The example configuration for the Telegram_bot integration described in the Telegram (notify) integration uses the polling method.
https://www.home-assistant.io/integrations/telegram#configuration

You can also configure the Telegram_bot integration to use the webhook method.
https://www.home-assistant.io/integrations/telegram_bot/

Comment (frenck) [reply source/_integrations/telegram.markdown:7]: Right, that makes it polling. Thanks for clarifying! 👍

**Review by frenck — APPROVED** [review]
Thanks, @milanmeu 👍 

Happy Hacktoberfest 🎉

**Pull request merged**